natgeo/mortar
# [SYTO-916] Playlist: Video does not load on FF

https://jira.natgeo.com/browse/SYTO-916

Just for check on FF and Edge.
The implemented approach was different in order to keep the layout consistency among the browsers with the video description and the video.

#### How to test
1. ``gulp serve``
2. Open /components
3. Scroll down until the video playlist section
4. The video into the VideoPlaylist should load properly with aspect ratio 16:9.
